extensions/parquet/table/src/main/java/io/deephaven/parquet/table/util/TransferUtils.java

```diff
@@ -0,0 +1,66 @@
+package io.deephaven.parquet.table.util;
+
+import io.deephaven.time.DateTimeUtils;
+import io.deephaven.util.QueryConstants;
+import org.jetbrains.annotations.Nullable;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+
+/**
+ * Utility methods for transferring data between Deephaven and Parquet.
+ */
+public class TransferUtils {
+    /**
+     * Returns nanoseconds from the Epoch for a {@link LocalDateTime} value in UTC timezone.
+     *
+     * @param localDateTime the local date time to compute the Epoch offset for
+     * @return nanoseconds since Epoch, or a NULL_LONG value if the local date time is null
+     */
+    public static long epochNanosUTC(@Nullable final LocalDateTime localDateTime) {
+        if (localDateTime == null) {
+            return QueryConstants.NULL_LONG;
+        }
+        return DateTimeUtils.epochNanos(localDateTime.atZone(ZoneId.of("UTC")));
```

Comment:
Earlier this was done as
```        
return TimeUnit.SECONDS.toNanos(localDateTime.toEpochSecond(ZoneOffset.UTC))
                + localDateTime.toLocalTime().getNano();
```
Chip suggested the new approach.

Reply:
No, this new approach is worse; it causes us to lookup the time zone (worse), allocate a ZonedDateTime (much worse), repeat the null check (slightly worse), etc.

The only argument in favor of using the new method is some overflow testing it does. I'm willing for us to do that overflow testing if we can make the methods or constants public. We could also just use `Math.addExact`.

Actually, we could use `DateTimeUtils.secondsToNanos()` to help; if we do that, we're only duplicating the null check. Making a suggestion accordingly.